src/components/FavoritesBtn.tsx

```diff
@@ -6,30 +6,40 @@ import '../styles/FavoritesBtn.scss';
 
 type FavoritesBtnProps = {
   movieId: number;
+  isMovieIdInFavorites?: boolean;
 };
 
-const FavoritesBtn: React.FC<FavoritesBtnProps> = ({ movieId }) => {
+const FavoritesBtn: React.FC<FavoritesBtnProps> = ({
+  movieId,
+  isMovieIdInFavorites
```

Comment:
isnt this same as line 16? what the difference?

Reply:
After your comment, I realized I gave a misleading name to a state in the first place.  isMovieIdInFavorites here is came from Context and in charge of changing a favorite icon state when a user clicks a star rating icon (when clicking a 5)